Gemfile

```diff
@@ -25,7 +25,7 @@ gem "turbo-rails"
 gem "stimulus-rails"
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
-gem "jbuilder"
+# gem "jbuilder"
```

Comment:
jbuilder is the default JSON rendering package for rails. It's OK though tends to be slow so not great with large objects. I've disabled it for now, but it is something we can easily add in later if needed.